wiki/Welcome/vi.md

```diff
@@ -0,0 +1,47 @@
+# Chào mừng
+
+Chào mừng đên osu!, một trò chơi nhịp điệu được lập trình bởi peppy với 4 chế độ chơi: osu!, chế độ bấm vòng tròn; osu!taiko, chế độ đánh trống; osu!catch, chế độ hứng quả; và osu!mania, chế độ bấm phím.
+
+## Một vài điều cơ bản
+
+- [Cài đặt](/wiki/Installation)
+- [Đăng kí](/wiki/Registration)
+- [Luật lệ](/wiki/Rules)
+- [FAQ](/wiki/FAQ)
+- [Chế độ chơi](/wiki/Game_mode)
+  - ![osu! mode icon](/wiki/shared/mode/osu.png) [osu!](/wiki/Game_mode/osu!)
+  - ![osu!taiko icon](/wiki/shared/mode/taiko.png) [osu!taiko](/wiki/Game_mode/osu!taiko)
+  - ![osu!catch icon](/wiki/shared/mode/catch.png) [osu!catch](/wiki/Game_mode/osu!catch)
+  - ![osu!mania icon](/wiki/shared/mode/mania.png) [osu!mania](/wiki/Game_mode/osu!mania)
+- [Chế độ nhiều người chơi](/wiki/Multi)
+- [Tiêu chuẩn xếp hạng](/wiki/Ranking_Criteria)
+  - ![osu! icon](/wiki/shared/mode/osu.png) [osu!](/wiki/Ranking_Criteria/osu!)
+  - ![osu!taiko icon](/wiki/shared/mode/taiko.png) [osu!taiko](/wiki/Ranking_Criteria/osu!taiko)
+  - ![osu!catch icon](/wiki/shared/mode/catch.png) [osu!catch](/wiki/Ranking_Criteria/osu!catch)
+  - ![osu!mania icon](/wiki/shared/mode/mania.png) [osu!mania](/wiki/Ranking_Criteria/osu!mania)
+
+## Beatmapping
+
+*Trang chính: [Beatmapping](/wiki/Beatmapping)*
+
+Beatmapping là quá trình của một người sáng tạo tạo ra beatmap. Quá trình này bao gồm: chọn nhạc, căn thời gian và tạo ra beatmap, và cuối cùng là thử beatmap. Nó cũng có thể bao gồm: thêm một video, thêm một [storyboard](/wiki/Storyboarding), và/hoặc thêm một [skin tự chọn](/wiki/Skinning). Tất cả những điều trên có thể được làm bởi cá nhân người sáng tạo hoặc với những người dùng khác, nhiều khi được gọi là *storyboarder*, *skinners*, và/hoặc *người sáng tạo được mời* dựa trên những điều họ làm. 
```

Comment:
```suggestion
Beatmapping là quá trình của một người sáng tạo làm ra beatmap. Quá trình này bao gồm: chọn nhạc, căn thời gian và làm ra beatmap, và cuối cùng là thử beatmap. Nó cũng có thể bao gồm: thêm một video, thêm một [storyboard](/wiki/Storyboarding), và/hoặc thêm một [skin tự chọn](/wiki/Skinning). Tất cả những điều trên có thể được làm bởi cá nhân người sáng tạo hoặc với những người dùng khác, nhiều khi được gọi là *người làm storyboard*, *người làm skin*, và/hoặc *người sáng tạo được mời* dựa trên những điều họ làm. 
```
Replace with a synonym (`làm`) of `tạo`, `sáng tạo tạo` makes it duplicate and sound weird.
And Vietnamese doesn't have suffix, so you should translate the `er` to `person who do ...`, of course I don't find this is an enforcement, you can keep that as is so it feels natural (by not translating the terminologies), at which the community at large already feels familiar with them.

Reply:
Well, on the other hands, it would state the needs to change `Beatmapping` and `Skinning`, ...etc to `Làm beatmap`, `Làm skin` though.
So you may keep it.

Reply:
Resolved in [846fe6a](https://github.com/ppy/osu-wiki/pull/5953/commits/846fe6ab46d9e9042496f546d54ba0aad6723a89)